mmdeploy/codebase/mmpose/deploy/pose_detection_model.py

```diff
@@ -136,6 +150,74 @@ def show_result(self,
             win_name=win_name)
 
 
+@__BACKEND_MODEL.register_module('sdk')
+class SDKEnd2EndModel(End2EndModel):
+    """SDK inference class, converts SDK output to mmcls format."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def _cs2xyxy(self,
+                 _center: np.ndarray,
+                 _scale: np.ndarray,
+                 padding: float = 1.25):
+        """This encodes bbox(x,y,w,h) into (center, scale)
+
+        Args:
+            _center: (np.ndarray[float32](2,)) Center of the bbox (x, y)
+            _scale: (np.ndarray[float32](2,)) Scale of the bbox w & h
+
+        Returns:
+            - np.ndarray[float32](2,): fake box if keypoint, the process in
+                topdown_affine will calculate original center, scale.
+        """
+        scale = _scale.copy()
+        scale = scale / padding * 200
+        center = _center.copy()
+        # fake box
+        box = np.array([center - 0.5 * scale,
+                        center + 0.5 * scale - 1]).flatten()
+        return box
+
+    def forward(self, img: List[torch.Tensor], *args, **kwargs) -> list:
```

Comment:
Please add more detail docstring about why do we need a new forward

Reply:
done